analyses/stackanalyses/version_controller.go

```diff
@@ -0,0 +1,10 @@
+package stackanalyses
```

Comment:
no need to have one more level of indirection just for version. shall be called directly.

version doesn't have any correlation with sa too. don't complicate.

Reply:
ack.